src/models/subgraph-entities/vX/protocol-log.ts

```diff
@@ -8,6 +8,8 @@ export type ProtocolLog = {
   paymentsCount: number
   redeemCount: number
   erc20Count: number
+  trendingLastUpdatedTimestamp: number
+  oldestTrendingPayEvent: string
```

Comment:
these are primarily helper properties used by the subgraph itself. probably not needed by the UI